pytorch_lightning/trainer/trainer.py

```diff
@@ -279,6 +403,19 @@ def fit(
         val_dataloaders: Optional[Union[DataLoader, List[DataLoader]]] = None,
         datamodule: Optional[LightningDataModule] = None,
     ):
+        r"""
+        Runs the full optimization routine.
+
+        Args:
+            model: Model to fit.
+
+            train_dataloader: A Pytorch DataLoader with training samples. If the model has
+                a predefined train_dataloader method this will be skipped.
+
+            val_dataloaders: Either a single Pytorch Dataloader or a list of them, specifying validation samples.
+                If the model has a predefined val_dataloaders method this will be skipped
+
```

Comment:
`datamodule` seems to missing in the arg description
```suggestion
datamodule: A instance of :class:`LightningDataModule`.
```

Reply:
can we find a more meaningful description for this?